src/lib/frontend/components/CustomAppBar.svelte

```diff
@@ -69,76 +65,62 @@
-                    on:click={toggleOverlaySearchBarVisibility}
-            >
-                {#if (isOverlaySearchBarVisible)}
-                    <CloseSearchIcon/>
-                {:else }
-                    <ExpandSearchIcon/>
-                {/if}
-            </button>
+        <div class='flex w-full justify-end items-center gap-6 py-3'>
             {#if (!isOnHomePage)}
-                <button class='text-lg hidden sm:hidden md:block font-thin' on:click={navigateToHomePage}>
+                <button class='text-sm hidden sm:hidden md:block font-semibold' on:click={navigateToHomePage}>
                     Accueil
                 </button>
             {/if}
-            <button class='text-lg hidden sm:hidden md:block font-thin' on:click={navigateToContactUsPage}>
+            <button class='text-sm hidden sm:hidden md:block font-semibold mr-4' on:click={navigateToContactUsPage}>
                 Contactez-nous
             </button>
             {#if (!isOnHomePage)}
                 <div class='text-3xl relative flex md:hidden'>
                     <button
-                            class='hover:text-orange-500 text-3xl flex justify-center items-center'
+                            class='hover:text-orange-500 text-sm flex justify-center items-center'
                             data-testid="home-button"
                             on:click={navigateToHomePage}>
                         <HomeIcon/>
                     </button>
                 </div>
             {/if}
             <div class='text-3xl flex relative'>
-                <button
-                        class='hover:text-orange-500'
-                        on:click={navigateToShoppingCartPage}
-                >
-                    <CartIcon/>
+                <button class='hover:text-orange-500' on:click={navigateToShoppingCartPage}>
+                    <img alt="remove" class="h-5 w-5" src="shopping-cart.svg">
                 </button>
-                <span class='absolute left-5 bottom-5 bg-orange-500 text-xs text-white w-4 h-4 flex justify-center items-center rounded-full border border-orange-500'>
+
+                <span class='absolute left-4 bottom-4 bg-red text-10 text-white w-4 h-4 flex justify-center items-center rounded-full border border-orange-500'>
                   {numberOfProductsInCart}
                 </span>
             </div>
-            <div class='text-3xl relative hidden sm:hidden md:block'>
+
+            <div class='relative hidden sm:hidden md:block'>
                 <button
-                        class='hover:text-orange-500 text-2xl flex justify-center items-center'
+                        class='hover:text-orange-500 flex justify-center items-center'
                         data-testid="user-profile-button"
                         on:click={onUserProfileClicked}>
-                    <UserProfileIcon/>
+                    <img alt="remove" class="h-5 w-5 ml-2" src="user.svg">
```

Comment:
Why img instead of Svelte Icon components? Can't we customize the assets used by the UserProfileIcon? Or at least then create a component encapsulating the custom <img>

Reply:
I know there is a way to create a component in React with svg content, i think there should be a way to encapsulate the svg in a component